cl/scrapers/management/commands/cl_scrape_opinions.py

```diff
@@ -35,23 +35,35 @@
 die_now = False
 cnt = CaseNameTweaker()
 
+VALID_CITE_TYPES = {
+    "federal": 1,
+    "neutral": 8,
+    "scotus_early": 5,
+    "specialty": 4,
+    "specialty_west": 7,
+    "specialty_lexis": 6,
+    "state": 2,
+    "state_regional": 3,
```

Comment:
Rather than duplicating the values in the `CITATION_TYPES` variable in `cl.search.models.Citation`, we should factor it out of the class so the values can be used in multiple places.

Reply:
yes of course.